tomni/transformers/json2bbox/main.py

```diff
@@ -30,17 +33,16 @@ def json2bbox(scf_object: dict) -> tuple:
                 int(round(max(x_values))),
                 int(round(max(y_values))),
             )
-        elif scf_object["type"] in ["ellipse", "circle"]:
+        elif scf_object["type"] in ["circle", "ellipse"]:
             # Assumes that angleOfRotation. This will be changed in the future.
```

Comment:
assumes that angelofrotation what

Reply:
not sure I think it was there already, it's related to how before the angle of rotation was not taken into account but I think it can be removed now.